code/modules/atmospherics/machinery/pipes/layered.dm

```diff
@@ -0,0 +1,315 @@
+/*
+Layered pipes - Same as the simple pipes, but includes definitions for the other two layers so mappers in the future don't want to kill themselves
+*/
+
+/*=====The big Kahunas=====*/
+
+//Supply 
+
+/obj/machinery/atmospherics/pipe/simple/supply/visible/bottom
+	layer = 2.3
+	piping_layer = 1
```

Comment:
The original PR had defines for these, please use.

Reply:
Are you referring to GAS_PIPE_VISIBLE_LAYER?

Reply:
`#define PIPING_LAYER_MIN 1`
`#define PIPING_LAYER_MAX 3`

Reply:
Re-thinking, they actually aren't for each one, they're a min/max incase we increase the number of layerable pipes in future.

This is actually fine, but I was referring to
`#define PIPING_LAYER_MIN 1`
`#define PIPING_LAYER_MAX 3`

Reply:
Alright - I had double checked - I only gave them hard-coded definitions for the mappers benefit (they'll only exist in that one state), but if there's ever a define I'll happily include it.

Reply:
Aye, I jumped the gun. Lets get those unnecessary level stuff cleaned up and this can merge imo.